homeworks/zlinszkygyorgy/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,13 @@
+name = input("What is your name:")
+age = input("What is your age:")
+experience = input("Your python experience in years:")
+developer = input("Do you want to be a Python developer? (yes/no)")
+
+name = name.strip().capitalize()
```

Comment:
Nem hiba, de ha szeretnéd, már az első soron is meghívhatod a metódusokat:
name = input("What is your name:").strip().capitalize()